addons/website_sale_picking/models/sale_order.py

```diff
@@ -0,0 +1,19 @@
+from odoo import models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def _check_carrier_quotation(self, force_carrier_id=None, keep_carrier=False):
+        res = super()._check_carrier_quotation(force_carrier_id, keep_carrier)
```

Comment:
Always pass positional args as positional args, and kwargs as kwargs.
```suggestion
        res = super()._check_carrier_quotation(force_carrier_id=force_carrier_id, keep_carrier=keep_carrier)
```